src/tabs/tab-header.component.ts

```diff
@@ -80,6 +82,11 @@ export class TabHeader {
 	@ViewChild("tabItem", { static: false }) tabItem: ElementRef;
 
 	protected _cacheActive = false;
+	protected title = "Tab header";
```

Comment:
We should probably expose this as an `Input`, since the actual `title` may not always be 1-1 with the `textContent`